avocado/utils/external/spark.py

```diff
@@ -479,7 +479,7 @@ def add(self, input_set, item, i=None, predecessor=None, causal=None):
     def makeSet(self, token, sets, i):
         cur, next_item = sets[i], sets[i + 1]
 
-        ttype = token is not None and self.typestring(token) or None
+        ttype = token is not None and self.typestring(token) or None  # pylint: disable=R1709
```

Comment:
Hi @ana, my current pylint, is not raising any warning on this file. Could you please describe the current version you are using and why this is needed? Any CI using a modern pylint? Should we bump the pylint version requirement on requirements-dev.txt ?